openadmet/models/tests/unit/comparison/test_comparison.py

```diff
@@ -17,6 +17,45 @@ def test_get_comparison_class():
         get_comparison_class("NotARealClass")
 
 
+def test_posthoc_fails_on_incorrect_inputs():
+    comp_obj = PostHocComparison()
+    with pytest.raises(ValueError):
```

Comment:
Use `with pytest.raises(ValueError, match="XXYYZZ"):` to be specific about what error you are trying to catch. Here and all instances below.